### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by aster-alemu**
In this PR, I have created the following:-

- Added a .gitignore file and added test.md.
- Set up linters for HTML and CSS.
- Added descriptive README file to the project.
- Added 3 files (each one in a separate commit). i.e. index.html, style.css and test.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Great thanks to one of the Micronat Tarikwa Tesfa she helps me a lot
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (ShoiraTa) [inline README.md]: - [ ] `MIT` link is broken, please consider adding `MIT` License file in the project in order to fix the link.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @aster-alemu ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

### OPTIONAL
- [ ] OPTIONAL: When committing please use the imperative mood in the subject line and do not forget to capitalise sentences.  
```
Added text header => Add text header
created test.md file => Create test.md file
```

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (micheaol) [reply README.md]: As requested by the previous code reviewer, MIT link is broken, please consider adding the MIT License file in the project in order to fix the link.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
-# HelloMicroverse
-This is a hello microverse week one project
+# Hello Microverse
+
+> This is the first project at Microverse first week activity.
+
```
Comment (micheaol) [inline README.md:4]: Please ensure that Linter should be set up correctly. Linter result should be green, this means that the last commit has a green checkmark (✅) or a summary of the past checks is in the PR.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @aster-alemu , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 3]
```diff
@@ -1,2 +1,34 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Great thanks to one of the Micronat Tarikwa Tesfa she helps me a lot
+
+## 📝 License
+
+This project is not licensed
```
Comment (julie-ify) [inline README.md:34]: - [ ] **[OPTIONAL]** You can add an `MIT` license to your project just by creating a `MIT.md` file in the root directory of your project, that is, the same directory as your `README.md` file. You then copy the `MIT.md` file from this [repository](https://github.com/microverseinc/readme-template/blob/master/MIT.md) into your own `MIT.md` file. Remember to personalise it to your `NAME` and `APP TYPE`. Finally, push your code to GitHub and you will have your MIT licence working 💯 👍

**Review by julie-ify — APPROVED** [review]
Hi @aster-alemu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**